### PR TITLE
coreos-koji-tagger: add support for overrides and multi-arch lockfiles

### PR DESCRIPTION
**Pull request by dustymabe**
Add support for the other lockfiles and a few other fixups:

- coreos-koji-tagger: move commit var access lower
- coreos-koji-tagger: Update README with more staging docs
- coreos-koji-tagger: add support for other architectures
- coreos-koji-tagger: support overrides files
- coreos-koji-tagger: remove koji 1.18 workaround

### CODE REVIEW

File: coreos-koji-tagger/coreos_koji_tagger.py [hunk 1]
```diff
@@ -271,13 +271,17 @@ def process(self, message: fedora_messaging.api.Message):
...
-        r = requests.get(url)
-
+        desiredrpms = set()
+        for arch in ['x86_64', 'aarch64', 'ppc64le', 's390x']:
+            for lockfile in ['manifest-lock']:
+                url = f'https://raw.githubusercontent.com/{repo}/{commit}/{lockfile}.{arch}.json'
+                logger.info(f'Attempting to retrieve data from {url}')
+                r = requests.get(url)
+                # parse the lockfile and add the set of rpm NEVRAs (strings)
+                if r.ok:
```
Comment (cgwalters) [inline coreos-koji-tagger/coreos_koji_tagger.py]: I think we should differentiate between `404` and any other error.  At least log non-404.

Comment (dustymabe) [reply coreos-koji-tagger/coreos_koji_tagger.py]: Yeah Let's log errors. Really even 404 shouldn't happen once we have all arches in place, so let's log that one too.

Comment (dustymabe) [reply coreos-koji-tagger/coreos_koji_tagger.py]: should be fixed now

File: coreos-koji-tagger/coreos_koji_tagger.py [hunk 2]
```diff
@@ -274,6 +273,7 @@ def process(self, message: fedora_messaging.api.Message):
 
         # Now grab data from the commit we should operate on:
         # XXX: should update for multi-arch
+        commit = msg['head_commit']['id']
```
[the remark this answers is not in the document]
Comment (dustymabe) [reply coreos-koji-tagger/coreos_koji_tagger.py]: converted it to a safe getter!

[the remark this answers is not in the document]
Comment (dustymabe) [reply coreos-koji-tagger/coreos_koji_tagger.py]: and then I converted it back.. there are messages with `head_commit` set to null so now I'm checking that first and then pulling `id` afterwards.

**Pull request merged**